views/default/forms/useradd.php

```diff
@@ -23,62 +25,58 @@
 	$admin = array_shift($admin);
 }
 
-?>
-<div>
-	<label><?php echo elgg_echo('name');?></label><br />
-	<?php
-	echo elgg_view('input/text', array(
-		'name' => 'name',
-		'value' => $name,
-	));
-	?>
-</div>
-<div>
-	<label><?php echo elgg_echo('username'); ?></label><br />
-	<?php
-	echo elgg_view('input/text', array(
-		'name' => 'username',
-		'value' => $username,
-	));
-	?>
-</div>
-<div>
-	<label><?php echo elgg_echo('email'); ?></label><br />
-	<?php
-	echo elgg_view('input/text', array(
-		'name' => 'email',
-		'value' => $email,
-	));
-	?>
-</div>
-<div>
-	<label><?php echo elgg_echo('password'); ?></label><br />
-	<?php
-	echo elgg_view('input/password', array(
-		'name' => 'password',
-		'value' => $password,
-	));
-	?>
-</div>
-<div>
-	<label><?php echo elgg_echo('passwordagain'); ?></label><br />
-	<?php
-	echo elgg_view('input/password', array(
-		'name' => 'password2',
-		'value' => $password2,
-	));
-	?>
-</div>
-<div>
-<?php 
-	echo elgg_view('input/checkboxes', array(
-		'name' => "admin",
-		'options' => array(elgg_echo('admin_option') => 1),
-		'value' => $admin,
-	));
-?>
-</div>
+echo elgg_view_input('text', [
+	'name' => 'name',
+	'value' => $name,
+	'label' => elgg_echo('name'),
+	'required' => true,
+]);
+
+echo elgg_view_input('text', [
+	'name' => 'username',
+	'value' => $username,
+	'label' => elgg_echo('username'),
+	'required' => true,
+]);
+
+echo elgg_view_input('email', [
+	'name' => 'email',
+	'value' => $email,
+	'label' => elgg_echo('email'),
+	'required' => true,
+]);
+
+echo elgg_view_input('checkbox', array(
+	'name' => 'autogen_password',
+	'value' => 1,
+	'default' => false,
+	'label' => elgg_echo('autogen_password_option'),
+	'checked' => false, // not using sticky value to make JS implementation cleaner
```

Comment:
Actually you don't need to set default, because the hidden input can't emit a change event anyway